CreationalDesignPatterns/AbstractFactory/CoffeeFactory/DoubleCoffee.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace CoffeeFactory
+{
+    public class DoubleCoffee : PlainCoffee

```

Comment:
Can we use lambda function for all the getters and extract all the magic strings and numbers in constatns

Reply:
We can do this for all the constants if you want in a separate project library instead of constants for each pattern?

Reply:
I believe it is better to have constants for each of the pattern but a seperate common project with constatns also will be fine